buildpipeline/pipelinejobs.groovy

```diff
@@ -14,10 +14,10 @@ def branch = GithubBranchName
 // will have a trigger that can be
 // **************************
 
-def alpine36Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/alpine.3.6.groovy')
+def linmuslPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/linux-musl.groovy')
```

Comment:
Can this be `linuxmuslPipeline`?  That matches what we have in core-setup.

Reply:
lol - I did this to match what is corefx because corefx and I did it in corefx to because it already had a linPipeline. I don't have a strong feeling either way but I also don't think it matters much.